docs/routes.json

```
@@ -150,6 +150,11 @@
         "title": "Next.js starter kit",
         "path": "/guides/nextjs-starter-kit",
         "hidden": true
+      },
+      {
+        "title": "Schema Validation",
```

```suggestion
        "title": "Schema validation",
```

No camel case, sentence case instead!